pkg/proxy/ipvs/proxier.go

```diff
@@ -589,6 +605,22 @@ func cleanupIptablesLeftovers(ipt utiliptables.Interface) (encounteredError bool
 			}
 		}
 	}
+	// Flush and remove all of our chains "-t filter" chains.
```

Comment:
Nit: Flush and remove all our "-t filter" chains.

Reply:
Done.